readme.md

```diff
@@ -5,7 +5,7 @@
 
 ## Install
 
-```
+```sh
```

Comment:
Can't use `sh` here as it's not a valid shell command because of the `$`

Reply:
Ah, sorry about that 👌